test/framework/deployments/kic/kubernetes.go

```diff
@@ -65,11 +65,14 @@ func (t *k8sDeployment) Deploy(cluster framework.Cluster) error {
 		return errors.Errorf("counting KIC pods. Got: %d. Expected: 1", len(pods))
 	}
 
-	k8s.WaitUntilPodAvailable(cluster.GetTesting(),
+	err = framework.WaitUntilPodAvailableE(cluster.GetTesting(),
 		cluster.GetKubectlOptions(t.ingressNamespace),
 		pods[0].Name,
 		framework.DefaultRetries,
 		framework.DefaultTimeout)
+	if err != nil {
```

Comment:
big nit, could just be `return framework.WaitUntilPod...`

Reply:
I'll change it in other PR as I want to push this one